docs/source/markdown/podman-image.1.md

```diff
@@ -30,10 +30,10 @@ The image command allows you to manage images
 | search   | [podman-search(1)](podman-search.1.md)              | Search a registry for an image.                                             |
 | sign     | [podman-image-sign(1)](podman-image-sign.1.md)      | Create a signature for an image.                                            |
 | tag      | [podman-tag(1)](podman-tag.1.md)                    | Add an additional name to a local image.                                    |
-| untag    | [podman-untag(1)](podman-untag.1.md)                | Removes one or more names from a locally-stored image.                      |
-| unmount   | [podman-image-unmount(1)](podman-image-unmount.1.md)  | Unmount an image's root filesystem.                                         |
 | tree     | [podman-image-tree(1)](podman-image-tree.1.md)      | Prints layer hierarchy of an image in a tree format.                        |
 | trust    | [podman-image-trust(1)](podman-image-trust.1.md)    | Manage container registry image trust policy.                               |
+| untag    | [podman-untag(1)](podman-untag.1.md)                | Removes one or more names from a locally-stored image.                      |
+| unmount   | [podman-image-unmount(1)](podman-image-unmount.1.md)  | Unmount an image's root filesystem.                                         |
```

Comment:
Are these supposed to be in alphabetical order? If so you might want to switch them.

Reply:
Yes.

Reply:
OK, I have a simple fix for `make xref_helpmsgs_manpages` that will (once this PR is merged) check that so it doesn't break again.

Reply:
Great.